views/surveys/view_survey_results.php

```diff
@@ -0,0 +1,259 @@
+<?php
+global $params, $route, $view;
+include ('../models/Notification.php');
+$pages = ['view','edit','create','delete','restore'];
```

Comment:
Change these values to reflect the pages you can actually go to. In your case it looks to only be `'results'`. If you are not using routing parameters at all, there is no need to have lines 4-15.